js/gateio.js

```diff
@@ -367,15 +367,8 @@ module.exports = class gateio extends Exchange {
                     'delivery': 'delivery',
                 },
                 'defaultType': 'spot',
-                'swap': {
-                    'fetchMarkets': {
-                        'settlementCurrencies': [ 'usdt', 'btc' ],
-                    },
-                },
-                'future': {
-                    'fetchMarkets': {
-                        'settlementCurrencies': [ 'usdt', 'btc' ],
-                    },
+                'derivates': {
```

Comment:
We will need to replace `derivatives` with `contract` (singular), that would be the unified common term to designate all derivatives. I think we could keep the per-type mapping tho.

Reply:
@kroitor  done 🙂